elitzur-avro/src/main/scala/com/spotify/elitzur/converters/avro/dynamic/dsl/AvroAccessorException.scala

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2021 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.spotify.elitzur.converters.avro.dynamic.dsl
+
+object AvroAccessorException {
+  class InvalidDynamicFieldException(msg: String) extends Exception(msg)
+
+  final val MISSING_TOKEN =
+    "Leading '.' missing in the arg. Please prepend '.' to the arg"
+
+  final val UNSUPPORTED_MAP_SCHEMA =
+    "Map schema not supported. Please use Magnolia version of Elitzur."
+
+  final val INVALID_UNION_SCHEMA =
+    "Union schemas containing more than one non-null schemas is not supported."
+
+  final val MISSING_ARRAY_TOKEN =
+    """
+      |Missing `[]` for an array field. Appending `[]` is required for all array fields except
```

Comment:
Still think we should change this messaging

Reply:
updated the messaging to be clearer